docs/pages/build/eas-builds-in-5-minutes.md

```diff
@@ -0,0 +1,165 @@
+
+So far, we've configured EAS Builds with `eas.json`. Now is the time to make the Android code buildable on our servers. Basically, the only thing you need to do is to provide your app's credentials to EAS Builds. As far as Android applications, you'll need a keystore, keystore password, key alias and key password to build the app binary.
+
+Expo CLI will help you generate the keystore. However, we need to tell `gradle` where it should look for the credentials.
+Open `android/app/build.gradle` and make the following changes:
+
+```diff
+diff --git a/android/app/build.gradle b/android/app/build.gradle
+index 980625b..4388b3b 100644
+--- a/android/app/build.gradle
++++ b/android/app/build.gradle
+@@ -152,15 +152,20 @@ android {
+             keyAlias 'androiddebugkey'
+             keyPassword 'android'
+         }
++        release {
++            def credentialsJson = new groovy.json.JsonSlurper().parse(rootProject.file("../credentials.json"))
++            storeFile rootProject.file("../" + credentialsJson[android"]["keystore"]["keystorePath"])
++            storePassword credentialsJson["android"]["keystore"]["keystorePassword"]
++            keyAlias credentialsJson["android"]["keystore"]["keyAlias"]
++            keyPassword credentialsJson["android"]["keystore"]["keyPassword"]
++        }
+     }
+     buildTypes {
+         debug {
+             signingConfig signingConfigs.debug
+         }
+         release {
+-            // Caution! In production, you need to generate your own keystore file.
+-            // see https://facebook.github.io/react-native/docs/signed-apk-android.
+-            signingConfig signingConfigs.debug
++            signingConfig signingConfigs.release
+             minifyEnabled enableProguardInReleaseBuilds
+             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
+         }
+```
+
+We're defining here a new signing config (`release`) and using it for the `release` build type. The code in the `release` signing config is reading the keystore configration from the file named `credentials.json`. You don't need to create this file manually. It'll be automatically created on our build servers and populated with the credentials you generate (or provide) with Expo CLI. Using your own `credentials.json` is an advanced feature of EAS Builds. You can learn more about it in the [Advanced Credentials](https://todo) page.
+
+After making the changes to the `build.gradle` file, commit them:
+
+- `git add android/app/build.gradle`
+- `git commit -m "Configure Android project"`
+
+<center><img src="/static/images/eas-builds/5-minute-tutorial/05-configure-android.png" /></center>
+
+### Running the Build
+
+Building the Android app binary is really straightforward. Just run `expo eas:build --platform android` and you'll be guided through generating or providing your own keystore. For the sake of simplicity of this tutorial, select `Generate new keystore` and hit `ENTER`.
```

Comment:
We should add the info here that if the user already had credentials set up on app with the same slug it will try to use them

Reply:
Done.